packages/about-you/composables/src/composables/getters/productGetters.ts

```diff
@@ -1,53 +1,60 @@
 /* istanbul ignore file */
-
+import { BapiProduct } from '@aboutyou/backbone';
 import {
   AgnosticMediaGalleryItem,
   AgnosticAttribute,
   AgnosticPrice,
   ProductGetters
 } from '@vue-storefront/core';
-import { BapiProduct } from '@aboutyou/backbone';
+import { getSettings } from '@vue-storefront/about-you-api';
 
 type ProductVariantFilters = any
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const getProductName = (product: BapiProduct): string => 'product-name';
 
-// platform doesn't seem to support slugs
 export const getProductSlug = (product: BapiProduct): string => product.id.toString();
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const getProductPrice = (product: BapiProduct): AgnosticPrice => {
+  const productPrices = product?.variants?.map(variant => variant.price?.withTax) || [0];
   return {
-    regular: 0,
-    special: 0
+    regular: Math.min(...productPrices),
+    special: Math.min(...productPrices)
   };
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getProductGallery = (product: BapiProduct): AgnosticMediaGalleryItem[] => [];
+export const getProductGallery = (product: BapiProduct): AgnosticMediaGalleryItem[] => {
+  return product?.images.map(imgObj => ({
+    big: `${getSettings().imgUrl}/${imgObj.hash}`,
+    normal: '',
+    small: ''
+  })) || [];
+};
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getProductCoverImage = (product: BapiProduct): string => '';
+export const getProductCoverImage = (product: BapiProduct): string => {
+  return `${getSettings().imgUrl}/${product.images[0].hash}`;
+};
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const getProductFiltered = (products: BapiProduct[], filters: ProductVariantFilters | any = {}): BapiProduct[] => {
   return products;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getProductAttributes = (products: BapiProduct[] | BapiProduct, filterByAttributeName?: string[]): Record<string, AgnosticAttribute | string> => {
-  return {};
+export const getProductAttributes = (product: BapiProduct, filterByAttributeName?: string[]): Record<string, AgnosticAttribute | string> => {
+  return Object.assign({}, ...filterByAttributeName.map(attrName => {
```

Comment:
you can simply return ` { ...filterByAttributeName.map(attrName => {})` w/o object.assign

Reply:
but `{... sth}` is equivalent to only `Object.assign({}, sth)`.

Reply:
That's true, it's just {... sth} is shorter :) Why using ES5 when u can use ES6